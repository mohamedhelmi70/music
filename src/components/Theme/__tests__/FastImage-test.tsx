import React from 'react';
import renderer from 'react-test-renderer';
import FastImage from '../FastImage';

// Describing a test suite
describe('<FastImage />', () => {
    test('Renders Fast Image correctly', () => {
        const tree = renderer.create(<FastImage url={'test.png'} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
