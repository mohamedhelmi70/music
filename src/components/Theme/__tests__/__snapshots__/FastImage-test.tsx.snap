// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FastImage /> Renders Fast Image correctly 1`] = `
<View
  style={
    Array [
      Object {
        "overflow": "hidden",
      },
      Object {},
    ]
  }
>
  <FastImageView
    resizeMode="cover"
    source={
      Object {
        "cache": "immutable",
        "headers": Object {
          "Authorization": "4f298a53e552283bee957836a529baec",
        },
        "priority": "normal",
        "uri": "test.png",
      }
    }
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  />
</View>
`;
